site/assets/styles/styles.css

```diff
@@ -27,10 +41,20 @@ a {
     float: left;
 }
 
+/*
+    distance:
+    Klasse für oberen Abstand 100px (genutzt in Hero)
```

Comment:
This is used in containers, heros have automatically a distance to top

Reply:
Resolved